devel.json

```
@@ -86,7 +86,10 @@
       ]
     },
     "ibm_cloud": {
-      "accounts": ["my_ic_account_one", "my_ic_account_two"]
+      "accounts": ["my_ic_account_one", "my_ic_account_two"],
+      "cluster_resources": {
+        "types": ["pods"]
+        }
```

Fix the indentation here to line things up correctly.

```suggestion
      "cluster_resources": {
        "types": ["pods"]
       }
```